### ISSUE TITLE
UI improvement for language-specific explanation

### ISSUE THREAD

**Issue by Geeyun-JY3** [WeblateOrg/weblate#9082]
Now that there could be a language-specific explanation, the UI label text should explicitly tell users that the explanation is language-specific. For example, `Explanation for Chinese (Simplified)`.
![image](https://user-images.githubusercontent.com/49396421/229286340-4bef1b4c-a18b-4ad1-819b-f5828ed42176.png)
Sometimes UI could have two “**Explanation**”. 

***

BTW, now that the glossary panel would show both the language-specific explanation and source language explanation (global explanation shown for all languages) while editing the corresponding target language, the current pattern for showing these explanations may confuse users. Users cannot distinguish the language-specific and global explanations in the current UI.
![image](https://user-images.githubusercontent.com/49396421/229287284-9456e8b1-fa6b-43f0-80ac-d2c534527179.png)

The layout of the glossary panel needs to be improved.
For reference, this is how Crowdin shows terms while both the source language and target language with **description (context)**:
![image](https://user-images.githubusercontent.com/49396421/229286826-0a54ca71-8dad-46d5-8704-212ff5bffc4a.png)

_Originally posted by @Geeyun-JY3 in https://github.com/WeblateOrg/weblate/issues/8810#issuecomment-1492952932_

**Comment by Lalman888** [comment]
@Geeyun-JY3  Hey,  Can I work on this issue ?

**Comment by nijel** [comment]
The tabular display, like we have now, makes it easy to find the terms. It got worse with the addition of glossary names, but with multiple explanations, it is getting really messy.

So, what we really should look at is how to render the glossary sidebar with all the features we have right now (multiple glossaries, edit link, forbidden/untranslatable terms display, explanations).

@Lalman888 if you want to come up with some UI mockups, that could be useful, but I don't think this issue is prepared for implementation.

PS: The discussed glossary entry is https://hosted.weblate.org/translate/f-droid/glossary-f-droid/zh_Hans/?checksum=6cf7cc0ddb514b1b

**Comment by Geeyun-JY3** [comment]
Why not change the UI label text of the language-specific explanation to something like `Explanation for Chinese (Simplified)` first?
![image](https://user-images.githubusercontent.com/49396421/229286340-4bef1b4c-a18b-4ad1-819b-f5828ed42176.png)
Two near “**Explanation**” are quite confusing.

**Comment by nijel** [comment]
Good point, that part is easy. I was talking about the right sidebar layout.

**Comment by Lalman888** [comment]
I think we can remove  sidebar and then make glossary model where glossary details will be available , In that way it will look more clean. Can we remove string information in sidebar or we can also add string info button and make model.

**Comment by nijel** [comment]
I believe this is addressed since cf8e73a6f7ecc6a765b3e9d7619f6f7e1b618b3a and b2b3646ba63.
